README.md

```diff
@@ -49,6 +49,7 @@ $ cdk destroy
 | [ecs-service-with-logging](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/ecs/ecs-service-with-logging/) | Starting a container fronted by a load balancer on ECS |
 | [fargate-service-with-logging](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/ecs/fargate-service-with-logging/) | Starting a container fronted by a load balancer on Fargate |
 | [custom-logical-names](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/custom-logical-names/) | Example of how to override logical name allocation |
+| [proxy-apigateway](https://github.com/aws-samples/aws-cdk-examples/tree/master/typescript/proxy-apigateway/) | Use ApiGateway to set up http proxy |
```

Comment:
Rename this 'http proxy apigateway' everywhere to disambiguate it from a lambda proxy.

Reply:
I don't understand what you mean if my thinking is corrected
Do you mean more specific about the difference between the'http proxy apigateway' and the'lambda proxy'?

If you don't mind could you see me some example?

Reply:
Change the name to 'http-proxy-apigateway'